erpnext/support/doctype/issue/issue.py

```diff
@@ -391,3 +398,32 @@ def get_holidays(holiday_list_name):
 	holiday_list = frappe.get_cached_doc("Holiday List", holiday_list_name)
 	holidays = [holiday.holiday_date for holiday in holiday_list.holidays]
 	return holidays
+
+
+@frappe.whitelist()
+def create_new_task_from_issue(self, assign_to):
+	self = frappe.json.loads(self)
+	doc = frappe.new_doc("Task")
+	doc.subject = self.get("subject")
+	doc.issue = self.get("name")
+	doc.description = self.get("description")
+	doc.priority = self.get("priority")
+	doc.status = self.get("status")
+	doc.project = self.get("project")
+	doc.save()
+	args = {
+		"assign_to": [assign_to],
+		"doctype": "Task",
+		"name": doc.name,
+	}
+	add(args)
+	frappe.msgprint("Task Created Successfully , {0}".format(get_link_to_form("Task", doc.name)))
+
+
+@frappe.whitelist()
+def get_if_task_exist(self):
+	self = frappe.json.loads(self)
+	if name := frappe.db.exists("Task", {"issue": self.get("name")}):
+		return name
+	else:
+		return
```

Comment:
```suggestion
def get_if_task_exist(issue):
	return bool(frappe.db.exists("Task", {"issue": issue}))
```

Why do you need this much code? This method can be removed entirely. There's `frappe.db.exist` on client side too.